### PR TITLE
feat(workflow): Remove semver flag from frontend

### PR DESCRIPTION
**Pull request by scttcper**
Feature has been in GA for a while

### CODE REVIEW

File: tests/js/spec/views/releases/list/index.spec.jsx [hunk 1]
```diff
@@ -186,7 +192,7 @@ describe('ReleasesList', function () {
       })
     );
 
-    expect(input.prop('value')).toBe('derp');
+    expect(input.prop('value')).toBe('derp ');
```
Comment (davidenwang) [inline tests/js/spec/views/releases/list/index.spec.jsx:195]: was the space at the end intentional?

**Review by davidenwang — APPROVED** [review]
lgtm

File: tests/js/spec/views/releases/list/index.spec.jsx (hunk 1 above)
Comment (scttcper) [reply tests/js/spec/views/releases/list/index.spec.jsx:195]: it seems to be coming from the newer search bar 🤷 automatically adds a space at the end.

Comment (davidenwang) [reply tests/js/spec/views/releases/list/index.spec.jsx:195]: ahhh gotcha

**Pull request merged**